### PR TITLE
correct doc for Series.quantile

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -3322,6 +3322,10 @@ def hpat_pandas_series_quantile(self, q=0.5, interpolation='linear'):
 
     Pandas API: pandas.Series.quantile
 
+    Limitations
+    -----------
+    Parameter interpolation is currently unsupported by Intel Scalable Dataframe Compiler
```
Comment (akharche) [inline sdc/datatypes/hpat_pandas_series_functions.py]:
```suggestion
    Parameter ``interpolation`` is currently unsupported.
```

Comment (densmirn) [reply sdc/datatypes/hpat_pandas_series_functions.py]: Actually this parameter is supported, but only with default value.

**Comment by Rubtsowa** [comment]
@densmirn  'Is numpy.absolute correct in seealso block?'
Yes, It is correct. https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.quantile.html

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 2]
```diff
@@ -3362,12 +3366,11 @@ def hpat_pandas_series_quantile(self, q=0.5, interpolation='linear'):
...
-    .. note::
-
-        Parameter interpolation is currently unsupported by Intel Scalable Dataframe Compiler
-
     .. seealso::
 
+        :ref:`core.window.Rolling.quantile <pandas.core.window.Rolling.quantile>`
+            Calculate the rolling quantile.
+
         `numpy.absolute <https://docs.scipy.org/doc/numpy/reference/generated/numpy.percentile.html#numpy.percentile>`_
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_series_functions.py]:
```suggestion
        `numpy.percentile <https://docs.scipy.org/doc/numpy/reference/generated/numpy.percentile.html#numpy.percentile>`_
```

File: sdc/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -3364,13 +3368,13 @@ def hpat_pandas_series_quantile(self, q=0.5, interpolation='linear'):
...
-        Parameter interpolation is currently unsupported by Intel Scalable Dataframe Compiler
-
     .. seealso::
 
-        `numpy.absolute <https://docs.scipy.org/doc/numpy/reference/generated/numpy.percentile.html#numpy.percentile>`_
+        :ref:`core.window.Rolling.quantile <pandas.core.window.Rolling.quantile>`
+            Calculate the rolling quantile.
+
+        `numpy.percentile
+        <https://docs.scipy.org/doc/numpy/reference/generated/numpy.percentile.html#numpy.percentile>`_
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_series_functions.py]: Please add some description for the link.

**Pull request merged**